harness/determined/common/api/authentication.py

```diff
@@ -215,6 +215,9 @@ def logout(
     if session_user is None:
         return
 
+    if session_user == token_store.get_active_user():
+        token_store.clear_active()
```

Comment:
Extra credit 2: The docstring for `TokenStore` isn't totally right in that it maintains _both_ a token store _and_ an "active user" string. Which... makes its name kind of misleading. Is a refactor wanted?

Reply:
updated the docstring.  No a refactor is not wanted, because there is only one auth.json file where this is stored, and so it makes sense that there is only one object to control it.